package logger

import "io"

// LogError записывает в out сообщение об ошибке err.
func LogError(err error, out io.Writer) {
	// Реализуй меня.
}
